cli/command/cli.go

```diff
@@ -82,6 +82,11 @@ type DockerCli struct {
 	dockerEndpoint     docker.Endpoint
 	contextStoreConfig store.Config
 	initTimeout        time.Duration
+
+	// baseCtx is the base context used for internal operations. In the future
+	// this may be replaced by explicitly passing a context to functions that
+	// need it.
+	baseCtx context.Context
```

Comment:
Should keep an eye on this and see if we can factor it out later 👀

Reply:
cc @cpuguy83